L1Trigger/TrackTrigger/interface/conifer.h

```diff
@@ -0,0 +1,157 @@
+#ifndef CONIFER_CPP_H__
```

Comment:
Maybe this is temporary, but I think this should be placed more centrally in cmssw so that all other subgroups can use it easily. I know Sioni was working on getting this done but I'm not certain what the status of the integration is.

Reply:
This is temporary, I am unsure of the best way of including this in CMSSW